languages/rust/oso/src/host/from_polar.rs

```diff
@@ -126,7 +127,7 @@ impl<T: FromPolar> FromPolar for Vec<T> {
 
 impl<T: FromPolar> FromPolar for HashMap<String, T> {
```

Comment:
Do avoid redundancy, could we instead have this implementation be something like:

```
impl <T: FromPolarValue> FromPolar for T {
    fn from_polar(term: &Term, _: &Host) -> {
        T::from_polar_value(PolarValue::from_term(term))
    }
}
```

Reply:
ah yeah that cleans it up a ton